src/web/lib/components/App/index.scss

```diff
@@ -24,6 +24,7 @@
     background-position: center center;
     background-repeat: no-repeat;
     background-size: 164px 164px;
+    min-height: 28px;
```

Comment:
Can you explain why you've chosen 28px? First you chose 22px, now 28px. I'd like to understand the reasoning behind it.

Reply:
because when I am running it on my local system, 22 px is not a perfect fit, somewhere it is collapsed with the banner but 28 px is a perfect fit on my screen, so, therefore, I have chosen this 28px